README.md

```diff
@@ -183,6 +183,7 @@ You can override the default styles and add new ones:
   You can also use `_variables_project.scss` to specify your own values for any 
   of the default 
   [Bootstrap 4 variables](https://getbootstrap.com/docs/4.0/getting-started/theming/).
+* You can update the Kubeflow example projects variables in the [website configuration file](https://github.com/hefedev/website/blob/7406b6f0b248d4881537cdb957dc5b27ae71c3a5/config.toml).
```

Comment:
This comment doesn't fit too well here, because this section of the README is about CSS styling. Also, the link points to the `hefedev` repo instead of the `kubeflow/website` repo.

See my comment below about keeping the content in the file corresponding to the page rather than in the config file.

Reply:
Agree 💯